<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BarnDetailsDialog</class>
 <widget class="QDialog" name="BarnDetailsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	font: 24px &quot;IRANSans Light&quot;;
	color: #494949;
}

#BarnDetailsDialog { background-image: url(:/img/barn-bg.png) }</string>
  </property>
  <widget class="QPushButton" name="lblUpgrade">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font: &quot;IRANSans Light&quot;;
	font-size: 21px;
	color: #425072;
	background-color: none;
	border: none;
	text-align: left;
}

QPushButton:hover{
	color: #29275f;
}</string>
   </property>
   <property name="text">
    <string>&gt; Upgrade</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblStorage">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>17</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Storage: 5/50</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblLevel">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>17</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Level: 1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblShovel">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>170</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>3</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblNail">
   <property name="geometry">
    <rect>
     <x>183</x>
     <y>170</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>3</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblEgg">
   <property name="geometry">
    <rect>
     <x>345</x>
     <y>170</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>3</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblAlfalfa">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>304</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>3</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblMilk">
   <property name="geometry">
    <rect>
     <x>183</x>
     <y>304</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>3</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblWool">
   <property name="geometry">
    <rect>
     <x>345</x>
     <y>304</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>3</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblUpgradeShovel">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>210</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>3/5</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblUpgradeNail">
   <property name="geometry">
    <rect>
     <x>194</x>
     <y>210</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>3/5</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblUpgradeTime">
   <property name="geometry">
    <rect>
     <x>351</x>
     <y>210</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>2 Days</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblUpgradeInfo">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>263</y>
     <width>450</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #3fa542;
</string>
   </property>
   <property name="text">
    <string>Max-storage will be 15</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUpgrade">
   <property name="geometry">
    <rect>
     <x>144</x>
     <y>313</y>
     <width>162</width>
     <height>49</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
font: 24px &quot;IRANSans Light&quot;;
	color: #494949;
}
QPushButton{
	border: none;
	color: #fff;
	background-image: url(:/img/upgrade-btn.png);
}
QPushButton:hover{
	background-image: url(:/img/upgrade-btn-hover.png);
}</string>
   </property>
   <property name="text">
    <string>2000</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblDaysLeft">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>80</y>
     <width>391</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18px &quot;IRANSans&quot;;
color: #818181;
background-image: url(:/img/clock_icon.png);
background-repeat: no-repeat;
padding-left: 28px;</string>
   </property>
   <property name="text">
    <string>2 Days Left</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
